### DISCUSSION TITLE
What is the right Way to configure the Nuxt app same as Sentry Documentation for vue?

### DISCUSSION THREAD

**Discussion by mahfuz-rahman007** [nuxt-community/sentry-module#685] [Q&A]
I am using sentry in my Nuxt app and using this module. I understand some basic configurations but getting confused by the overall documentation.

When I create a project in sentry and select vue I get some instruction on how to implement sentry.

1. yarn add @sentry/vue

2. `
  import Vue from "vue";
  import Router from "vue-router";
  import * as Sentry from "@sentry/vue";

  Vue.use(Router);

  const router = new Router({
    
  });

  Sentry.init({
    Vue,
    dsn: "",
    integrations: [
      Sentry.browserTracingIntegration(),
      Sentry.replayIntegration(),
    ],
    tracesSampleRate: 1.0,
    tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
    replaysSessionSampleRate: 0.1,
    replaysOnErrorSampleRate: 1.0,
  });

  new Vue({
    router,
    render: (h) => h(App),
  }).$mount("#app");`

4. npx @sentry/wizard@latest -i sourcemaps

so how can I make this full implementation in the nuxt app, especially how can I configure this in nuxt.config

**Comment by rchl** [comment]
Module auto-configures all that based on its own settings.

Check documentation:
- https://sentry.nuxtjs.org/guide/performance
- https://sentry.nuxtjs.org/guide/session-replay
